.github/workflows/python.yml

```diff
@@ -204,7 +201,7 @@ jobs:
           TEST_GROUP: ${{ matrix.instance }}
 
   test-py3-snuba:
-    name: 'py3 snuba [ignore fails]'
+    name: '[ignore fails] python3.6 backend (snuba)'
```

Comment:
Should we change this to unicode bold `[𝗜𝗚𝗡𝗢𝗥𝗘 𝗙𝗔𝗜𝗟𝗦]` lol

Reply:
lol